frontend/app/page.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
```

Comment:
generally where would "use client" make sense?  does this mean we would not do any ssr below this level? We dont use this in sprinto today, but would be useful to understand at what level we start making client side components.

Reply:
This line makes only this page render on the client rest pages by default are SSR. In Sprinto, we have made it CSR by default probably (need to learn how this works in sprinto).